### ISSUE TITLE
Angular App Deployment in Netlify with Gitlab CI

### ISSUE THREAD

**Issue by rene-shigolah** [section-engineering-education/engineering-education#4935]
#### Proposed title of the article
Angular App Deployment with Gitlab CI

#### Proposed article introduction
Using a continuous integration (CI) server for backend systems has been an established best practice for a very long time. As the complexity and the scope of modern web frontends increases, so does the need for professional development techniques. This includes working with test-driven development, CI servers, and automated processes.

Any kind of online application may be built and deployed using the Netlify platform (not just Angular).
Deploying your program quickly and easily is made a lot easier by using this method. Netlify is more than simply a platform for running demonstrations or examples while writing blogs or when I need to see some code in action (not just locally).

#### Key takeaways 
- Create your angular project on Github (could also be on bitbucket/GitLab)
- Login into Netlify, look for your repo, and set up the build options
- Deploy the new website created!

#### Article quality
So basically, my focus in this article is the “CI” part. We will build a simple angular app, using a frontend framework and we’ll also take a peek at writing unit test cases. Furthermore, my content will be beginner-friendly and my code snippets will be well explained.

#### References
N/A

**Comment by Kayere** [comment]
Hello @rene-shigolah - Please provide more information on your introduction.

**Comment by rene-shigolah** [comment]
hello @Kayere, I have updated the introduction please check.

**Comment by rene-shigolah** [comment]
hello @Kayere @ahmadmardeni1 and @WanjaMIKE  I have updated the introduction please check.

**Comment by Kayere** [comment]
Topic approved @rene-shigolah.
